x-pack/plugins/lens/public/indexpattern_datasource/field_item.test.tsx

```diff
@@ -227,31 +227,14 @@ describe('IndexPattern Field Item', () => {
     clickField(wrapper, 'bytes');
 
     expect(core.http.post).toHaveBeenCalledTimes(2);
-    expect(core.http.post).toHaveBeenLastCalledWith(`/api/lens/index_stats/1/field`, {
-      body: JSON.stringify({
-        dslQuery: {
-          bool: {
-            must: [],
-            filter: [
-              {
-                bool: {
-                  should: [{ match_phrase: { 'geo.src': 'US' } }],
-                  minimum_should_match: 1,
-                },
-              },
-              {
-                match: { phrase: { 'geo.dest': 'US' } },
-              },
-            ],
-            should: [],
-            must_not: [],
-          },
+    expect(core.http.post.mock.calls[0]).toMatchInlineSnapshot(`
+      Array [
+        "/api/lens/index_stats/1/field",
+        Object {
+          "body": "{\\"dslQuery\\":{\\"bool\\":{\\"must\\":[],\\"filter\\":[],\\"should\\":[],\\"must_not\\":[]}},\\"fromDate\\":\\"now-7d\\",\\"toDate\\":\\"now\\",\\"fieldName\\":\\"bytes\\"}",
```

Comment:
Seems like something broke here - the inline snapshot doesn't include the `geo.src: US` match. Didn't run the code